example/src/components/ArrowBoxExamples.js

```diff
@@ -3,139 +3,52 @@ import ArrowBox from '../../../src/components/ArrowBox';
 import Code from '../../../src/components/Code';
 import Colors from '../../../src/components/Colors';
 
-const directions = ['top', 'right', 'bottom', 'left'];
-
 export default React.createClass({
-  getInitialState() {
-    return {
-      box1Show: false,
-      box2Location: 0,  // 0, 1, 2, 3 ~= 'top', 'right', 'bottom', 'left'
-      box2Show: false,
-    };
-  },
-
-  onClick1() {
-    this.setState({
-      box1Show: !this.state.box1Show,
-    });
-  },
-
-  onClick2() {
-    this.setState({
-      box2Location: this.state.box2Show ? (this.state.box2Location + 1) % 4 : this.state.box2Location,
-      box2Show: !this.state.box2Show,
-    });
-  },
-
   render() {
     return (
       <section className="_full-width-row" id="ArrowBox">
         <div className="_container _container_large">
           <h2 className="base--h2">Arrow Box</h2>
           <div className="row">
-            <button
-              className="base--button arrow-box-button"
-              onClick={this.onClick1}
-              style={{ position: 'relative' }}
+            <div
+              style={{
+                position: 'relative',
+                width: '10rem',
+                margin: 'auto',
+                backgroundColor: Colors.gray_10,
+                display: 'block',
+                padding: '1rem',
+              }}
             >
-              Click me!
+              This is the element that the ArrowBox is anchored to.
               <ArrowBox
                 direction="top"
-                show={this.state.box1Show}
+                show
```

Comment:
I would say make this `show={true}` just to make it more clear. Otherwise, someone might believe that they need to remove the attribute to hide the ArrowBox.